components/Form/SimpleSlider.tsx

```diff
@@ -0,0 +1,43 @@
+import Slider from "@mui/material/Slider";
+import { useState } from "react";
+import colors from "~/config/themes/colors";
+
+interface Props {
+  start: number;
+  end: number;
+  initial?: number;
+  onChange: (newValue: number) => void;
+}
+
+const SimpleSlider = ({ start, end, initial = 0, onChange }: Props) => {
+  const [value, setValue] = useState<number>(initial);
+
+  const handleCommitChange = (event: Event, newValue: number | number[]) => {
+    setValue(newValue as number);
+    onChange(newValue as number);
+  };
+
+  const handleSliderChange = (event: Event, newValue: number | number[]) => {
+    setValue(newValue as number);
+  };
+
+  function valueLabelFormat(value: number) {
+    return value + "th percentile";
```

Comment:
Nit: This won't work for every percentile. 1st, 2nd, 3rd, 21st, 23rd, 24th etc.

Reply:
good call